wgs/wgs-gather-vcfs/wgs-gather-vcfs.wdl

```diff
@@ -0,0 +1,169 @@
+## wgs-gather-vcfs.wdl
+## 
+## This workflow runs the GATK's `GatherVcfsCloud` on a GCS directory containing sharded VCF files
```

Comment:
Please be sure that update to the WDL header comments are reflected in the README.md